exports/celo.json

```diff
@@ -2,6 +2,1522 @@
   "name": "celo",
   "chainId": "42220",
   "contracts": {
+    "LBPManager": {
```

Comment:
![image](https://user-images.githubusercontent.com/30693990/206473193-27ecddda-2c07-48a3-bd19-64215e5a7aa3.png)

both contracts are the same, aren't they?